src/library_webgpu.js

```diff
@@ -2619,9 +2619,7 @@ var LibraryWebGPU = {
   // WGPUSurface
 
   wgpuSurfaceGetPreferredFormat: function(surfaceId, adapterId) {
-    var context = WebGPU.mgrSurface.get(surfaceId);
-    var adapter = WebGPU.mgrAdapter.get(adapterId);
-    var format = context["getPreferredFormat"](adapter);
+    var format = navigator["gpu"]["getPreferredCanvasFormat"]();
```

Comment:
Would be good to teach closure compiler about these things so we can use normal accessors here.  Doesn't have to be in the PR, but going forward it would make this code more readable I think.

Reply:
I'll have to ask you about how to do that. We wrote this very cautiously because I don't exactly understand the rules of closure compiler / exactly when it's safe to do that. Is it just string based (so any member named 'gpu' anywhere in any code would get deminified), or does it somehow know the type of the parent object? For basically every other `["name"]` in this file it wouldn't know the type.